files/build_templates/swss.service.j2

```diff
@@ -21,6 +21,8 @@ ExecStartPre=/usr/bin/docker exec database redis-cli -n 2 FLUSHDB
 ExecStartPre=/etc/init.d/sxdkernel start
 ExecStartPre=/usr/bin/mst start
 ExecStartPre=/sbin/modprobe i2c-dev
+# i2c_ismt driver resides on the same bus as lpc_i2c, we remove it since its not used in Mellanox box to allow lpc_i2c to be inserted
+ExecStartPre=-/sbin/rmmod i2c_ismt
```

Comment:
no, other platform uses the driver, I believe. I still feel the right place should the hw_management package, in its init script, it can first check whether the module is loaded or not, then remove if it is loaded.